samples/SocketsSample/wwwroot/.gitignore

```diff
@@ -1 +1 @@
-js/
\ No newline at end of file
+lib/signalr-client
```

Comment:
If I unified the projects and put signalr in the same relative path we could move it to the common .gitignore.